crates/node-core/src/args/datadir_arg.rs

```diff
@@ -0,0 +1,33 @@
+//! clap [Args](clap::Args) for datadir config
+
+use clap::Args;
+use std::path::PathBuf;
+
+/// Parameters for database configuration
+#[derive(Debug, Args, PartialEq, Eq, Default, Clone)]
+#[command(next_help_heading = "Datadir")]
+pub struct DatadirArgs {
+    /// Set the absolute path to static files
+    #[arg(long = "datadir.static_files", help_heading = "Datadir", value_name = "PATH")]
+    pub static_files_path: Option<PathBuf>,
+}
```

Comment:
this should include the datadir argument, so all datadir-related args are in one place, wdyt @mattsse ?

Reply:
that makes sense yes

Reply:
a36025140292e8c959d0dce7cc76fff12ac7ed75